platform-coop-toolkit/pcc
# Added new template page "Events"

## Description

Added new events template page with tabs to show events filtered by event type

## Steps to test

1. Create a page and set the new template "Events"
2. Open the new page created
3. You should be able to see the events and navigate in tabs

**Expected behavior:** Show events page and the tabs navigation to filter events

## Additional information

The template page use ajax, then it's no necessary reload the page to change the page and tabs.
